packages/x-components/src/design-system/components/sliding-panel/default.scss

```diff
@@ -3,8 +3,8 @@
   background-color: var(--x-color-background-sliding-panel);
```

Comment:
the z index is not right:

![image](https://user-images.githubusercontent.com/6597815/126765805-45578bb4-c88f-4b85-9a6d-b0442f17d5d5.png)

Reply:
Same for hover:

![image](https://user-images.githubusercontent.com/6597815/126765847-65e7b299-f7c1-43a1-a11f-d0207b1287a5.png)

Reply:
Yep, `line 65` is what is breaking it:

```
&:not(.x-sliding-panel--no-gradient) {
    &::before,
    &::after {
      // layout
      position: absolute;
      z-index: 1;
      ...
```

Reply:
fixed with MORE z-index in the component